companies/zara-de.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
ZARA also has stores IRL, but this seems to handle it fine. Maybe adopt it for SportsDirect, too?